backend/app/usecases/chat.py

```diff
@@ -393,12 +464,16 @@ def propose_conversation_title(
     messages.append(new_message)
 
     # Invoke Bedrock
-    args = compose_args_for_anthropic_client(
+    args = compose_args(
         messages=messages,
         model=model,
     )
-    response = client.messages.create(**args)
-    reply_txt = response.content[0].text
+    if is_anthropic_model(args["model"]):
+        response = client.messages.create(**args)
```

Comment:
Like `compose_args`, the llm output generator method also should be abstract. Like

```py
response = generate_bedrock_response(arg, stream=True)
```

Reply:
(Same as other methods, please declare input and output types)

Reply:
The major difference comes from Anthorpic Client vs Bedrock Client model invocation, and streaming vs non-streaming (completion). Would suggest to have another PR if you want to refactor this.